conf/ocsci/default_config.yaml

```diff
@@ -52,12 +39,12 @@ REPORTING:
 # This is the default information about environment. Will be overwritten with
 # --cluster-conf file.yaml data you will pass to the pytest.
 ENV_DATA:
-  platform: '{{ DEFAULTS["DEPLOYMENT_PLATFORM"] }}'
   cluster_name: "ocs-ci"  # needs to be changed in ocscilib plugin
   cluster_namespace: 'openshift-storage'
-  region: '{{ DEFAULTS["AWS_REGION"] }}'
   # Do not change to specific version like v14.2.1-20190430 if not needed
   # cause we don't need to update it each time new 14.x version is released
   # but only once when move to new version like v15.
```

Comment:
Plese the comment here belong to line 49 for ceph image, so move this comment above this variable please